AdminClient/src/app/customers/dialog-body/dialog-body.component.html

```diff
@@ -0,0 +1,144 @@
+<div class="container" fxLayout
+     fxLayout.xs="column"
+     fxLayoutAlign="none"
+     fxLayoutGap="10px"
+     fxLayoutGap.xs="0">
+
+  <div id="create" fxFlex fxFlexOffset="50px" *ngIf="displayCreate()">
```

Comment:
Again see https://material.angular.io/components/input/overview to quickly give your inputs some nice styling that wont take long and can be considered an MVP. In my opinion having something presentable for an MVP is important as it otherwise distract the user from seeing the functionality @markscamilleri please share your input in this in case styling is not important for MVP.